website/docs/admin_server_installation.md

```diff
@@ -0,0 +1,29 @@
+---
+id: admin_server_installation
+title: Installing Ayon server 
+sidebar_label: Server installation
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+## Requirements
+
+- Docker with the compose plugin. To install the latest Docker, you can use this script: [https://get.docker.com](https://get.docker.com)
+- If you use a stand-alone `docker-compose` script instead of the compose plugin, make sure to use `docker-compose` wherever `docker compose` is used in this tutorial.
+- For production, using Linux is highly recommended. However, for evaluation purposes, Windows with WSL (Windows Subsystem for Linux) can be used.
+
+## Installation
+
+1. Clone this repository to your local machine.
+2. Tweak the `docker-compose.yml` file according to your requirements.
+3. You may use the `.env` file to set environment variables (for example, for SSO configuration).
+4. On Windows, comment-out or delete the `- "/etc/localtime:/etc/localtime:ro"` line from the `docker-compose.yml` file.
+5. Install addons to the `addons` directory.
+6. Modify the default settings in the `settings/template.json` file.
+7. Run the stack using `docker compose up -d`
```

Comment:
```suggestion
7. Run the stack using `docker compose up -d` (Unix) or `docker-compse up -d` (Windows).
```
I know you already mention is above but cant hurt to be explicit.

Reply:
Better to use linux and windows tabs for this. They can be synced in the whole page with the same tab setup as here https://github.com/ynput/ayon-documentation/blob/main/website/docs/dev_build.md?plain=1#L37-L252